tools/eval_net.py

```
@@ -458,6 +470,46 @@ def main(args):
         choices=("false", "standard", "sweep"),
         help="Type of noise evaluation",
     )
+
+    # Arguments for 2D Motion Corruption of the Dataset
+
+    parser.add_argument(
+        "--angle",
+        default=0,
+        type=float,
+        help=("How much rotation angle should be used for motion corruption " "of the dataset"),
+    )
+
+    parser.add_argument(
+        "--translation",
+        default=0,
+        type=float,
+        help=("How much translation should be used for motion " "corruption of the dataset"),
+    )
+    parser.add_argument(
+        "--nshots",
+        default=0,
+        type=int,
+        help=("How many shots should be used for motion corruption " "of the dataset."),
+    )
+    parser.add_argument(
+        "--trajectory",
+        default="blocked",
+        choice=("interleaved", "blocked"),
+        help=(
+            "Chooses between interleaved or blocked shots for motion " "corruption of the dataset"
+        ),
+    )
+
+    parser.add_argument(
+        "--two-dimensional",
+        default="true",
+        choice=("true", "false"),
+        help=("If true, MRI is 2D. Otherwise, MRI is 3D.")
+    )
+
```

this is actually dataset specific - fastMRI is 2D, mridata is 3D. Note your motion simulation does not support 3D yet, so i would actually just raise an error if it is not the fastMRI dataset you are using. or at least make sure to only use the fastmri dataset